src/components/common/RedactableValue.tsx

```diff
@@ -26,6 +26,8 @@ export interface RedactValueProps {
   onRedactionRequested?: () => void;
   editable?: boolean;
   iconRedact?: React.ReactElement;
+  redactedMessage?: string;
+  noValueMessage?: string;
```

Comment:
add a test for default and custom values?

Reply:
added!